micro-ui/web/micro-ui-internals/packages/react-components/src/atoms/svgindex.js

```
@@ -1108,10 +1108,10 @@ const DeleteIconv2 = ({styles, className, fill =COLOR_FILL,height="24",width="24
   </svg>
 );
 
-const InfoIconOutline = ({styles, className, fill = "#505A5F",height="20",width="20" }) => (
+const InfoIconOutline = ({ styles, className, fill = "#505A5F", height = "20", width = "20" }) => (
   <svg style={styles} className={className} xmlns="http://www.w3.org/2000/svg" width={width} height={height} viewBox="0 0 20 20" fill={fill}>
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)